compiler/passes/ResolveScope.cpp

```diff
@@ -851,6 +852,34 @@ Symbol* ResolveScope::lookupPublicImports(const char* name) const {
   return retval;
 }
 
+Symbol* ResolveScope::lookupPublicUnqualAccessSyms(const char* name,
+                                                   ModuleSymbol*& modArg) const {
+  UseImportList useImportList = mUseImportList;
+
+  for_vector_allowing_0s(VisibilityStmt, visStmt, useImportList) {
+    if (ImportStmt *is = toImportStmt(visStmt)) {
+      if (!is->isPrivate) {
+        for_vector(const char, unqualName, is->unqualified) {
+          if (astr(unqualName) == astr(name)) {
```

Comment:
I think the above two lines should be replaced with a call to the `skipSymbolSearch` method on ImportStmts (note that the method returns false aka "don't skip" when the name is in the unqualified symbol list).  That'll leave less code for me to update when I add the ability to rename symbols provided for unqualified access